client/src/pages/dashboard/EventTypes.js

```diff
@@ -32,54 +33,57 @@ const EventTypes = () => {
   const classes = useStyles();
   const theme = useTheme();
   const matches = useMediaQuery(theme.breakpoints.up("md"));
+
+  const userData = useContext(UserProvider.context);
+  
```

Comment:
```
const currentUser = userData.user.user;
```

that way we can call just 

```
{currentUser && (
...
```

Reply:
change to be addressed in branch associated to #66